js-simple-function/index.js

```diff
@@ -0,0 +1,38 @@
+export function func(testingString, firstLetter, secondLetter) {
+
+    // получилось два решения, одно - закомментированное
+    // второе - после того как узнал про метод lastIndexOf
+    // оба рабочи
+
+    // if(testingString.length <= 0) {
+    //     return -1;
+    // }
+
+    // let strArr = Array.from(testingString);
+    // let firstLetterLastIndex = 0;
+    // let secondLetterLastIndex = 0;
+
+    // strArr.forEach((letter, index) => {
+    //     if (letter === firstLetter) {
+    //         firstLetterLastIndex = index;
+    //     }
+    //     if (letter === secondLetter) {
+    //         secondLetterLastIndex = index;
+    //     }
+    // });
+
+    // if (firstLetterLastIndex === 0 && secondLetterLastIndex === 0) {
+    //     return -1;
+    // }
+
+    // return firstLetterLastIndex > secondLetterLastIndex ? firstLetterLastIndex : secondLetterLastIndex;
+
+    if (testingString.length === 0 | firstLetter?.length > 1 || secondLetter?.length > 1) {
```

Comment:
Отличная идея использовать optional chaining оператор, стоит добавить его в первое условие тоже
```suggestion
    if (testingString?.length === 0 | firstLetter?.length > 1 || secondLetter?.length > 1) {
```

Reply:
Интересно, что есть синтаксическая ошибка в условии.

Используется побитовое или `|`, скорее всего имеется в виду `||`, в нашем случае оператор вернет 1 или 0, которые кастуются в boolean как true и false.

Спасибо JS.